testsuite/elytron/src/test/resources/org/wildfly/test/security/common/elytron/jaas-login.config

```diff
@@ -0,0 +1,6 @@
+Entry1 {
+    org.wildfly.security.auth.TestLoginModule required;
```

Comment:
> `org.wildfly.security.auth`

It seems the content of this config file is not used as the module is in a different package.